.circleci/config.yml

```diff
@@ -1,6 +1,6 @@
 version: 2.1
 orbs:
-  architect: giantswarm/architect@1.1.1
+  architect: giantswarm/architect@dev:bump-app-build-suite-version
```

Comment:
TODO

- Merge giantswarm/app-build-suite#52 + make an app-build-suite release
- Release architect-orb with the new app-build-suite release
- Bump the version here